src/main/java/ai/elimu/web/ConnectionProviderWeb.java

```diff
@@ -0,0 +1,34 @@
+package ai.elimu.web;
+
+import ai.elimu.util.ConfigHelper;
+import org.hibernate.engine.jdbc.connections.spi.ConnectionProvider;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionProviderWeb implements ConnectionProvider {
+
+    @Override
+    public Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(ConfigHelper.getProperty("jdbc.url"), ConfigHelper.getProperty("jdbc.username"), ConfigHelper.getProperty("jdbc.password"));
```

Comment:
- good practice to pass required connection properties as an constructor arguments, this approach allow us to not couple `ConnectionProviderWeb` with an a singleton class `ConfigHelper` and makes code easier to support in future as we will control input dependencies of a class by constructor instead of unexpected behaviour in case of changed logic of `ConfigHelper` methods

Reply:
@nya-elimuai, yes

Reply:
☝🏽 @tomaszsmy